### PR TITLE
PWN-8192 - show error when fee is more then total in tokens

### PR DESCRIPTION
**Pull request by kabramovp2p**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-8192

## Description of Work

PWN-8192 - show error when fee is more then total in tokens

### CODE REVIEW

**Review by eduardmaximovich — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/bridge/send/statemachine/SendFeatureException.kt [hunk 1]
```diff
@@ -14,6 +14,10 @@ sealed class SendFeatureException(
         val invalidAmount: BigDecimal,
     ) : SendFeatureException()
 
+    data class FeeIsMoreThenTotal(
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/bridge/send/statemachine/SendFeatureException.kt:17]: `FeeIsMoreThanTotal`

File: app/src/main/java/org/p2p/wallet/bridge/send/statemachine/validator/SendBridgeValidator.kt [hunk 2]
```diff
@@ -18,13 +19,17 @@ class SendBridgeValidator {
...
-        val fees = fee.fee
-        // todo fees.networkFee, fees.messageAccountRent, fees.bridgeFee this fee is in SOL!
-        val feeList = listOf(fees.arbiterFee)
-        val feeSum: BigDecimal = feeList.sumOf { it.amountInToken }
+
+        val feeSum = getFeeSum(fee)
         if (feeSum.isMoreThan(inputAmount)) throw SendFeatureException.InsufficientFunds(inputAmount)
     }
 
+    fun validateIsFeeMoreThenTotal(state: SendState.Static, fee: SendFee.Bridge) {
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/bridge/send/statemachine/validator/SendBridgeValidator.kt:27]: `validateIsFeeMoreThanTotal`

File: app/src/main/java/org/p2p/wallet/bridge/send/ui/BridgeSendPresenter.kt [hunk 3]
```diff
@@ -194,6 +194,13 @@ class BridgeSendPresenter(
                                 sliderButton = null
                             )
                         }
+                        is SendFeatureException.FeeIsMoreThenTotal -> {
+                            setInputColor(R.color.text_rose)
+                            updateButtons(
+                                errorButton = TextContainer.Res(R.string.send_error_fee_more_then_amount),
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/bridge/send/ui/BridgeSendPresenter.kt:200]: `TextContainer(R.string.send_error_fee_more_then_amount)`

File: app/src/main/res/values/send_strings.xml [hunk 4]
```diff
@@ -75,7 +75,7 @@
     <string name="send_no_account_non_critical_continue">Continue with %s</string>
     <string name="send_no_account_non_critical_switch">Switch to another token</string>
 
-
+    <string name="send_error_fee_more_then_amount">The fee is more than the amount sent</string>
```
Comment (1sakov) [inline app/src/main/res/values/send_strings.xml:78]: `send_error_fee_more_than_amount`

**Review by 1sakov — APPROVED** [review]
minor comments

**Review by Davran331122 — APPROVED** [review]

**Pull request merged**